File: Back End/DVLD/DVLD_DataAccessLayer/Models/TestAppointment.cs
using System;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DVLD_DataAccessLayer.Models
{
    public class TestAppointment
    {
        public int TestAppointmentID { get; set; }
        public int LocalDrivingLicenseApplicationID { get; set; }
        public int TestTypeID { get; set; }
        public DateTime AppointmentDate { get; set; }
        public decimal PaidFees { get; set; }
        public int CreatedByUserID { get; set; }
        public bool IsLocked { get; set; }
        [AllowNull]
        public int? RetakeTestApplicationID { get; set; } = -1;
    }
}

File: Front End/DVLD/src/app/Models/TestAppointment.ts
export interface TestAppointment {
    testAppointmentID: number;
    localDrivingLicenseApplicationID: number;
    testTypeID: number;
    appointmentDate: Date; 
    paidFees: number;
    createdByUserID: number;
    isLocked: boolean;
    retakeTestApplicationID: number;
  }